mmdeploy/codebase/mmrotate/core/bbox/transforms.py

```diff
@@ -38,3 +38,65 @@ def poly2obb_le90__tensorrt(ctx, polys: torch.Tensor) -> torch.Tensor:
     width, _ = torch.max(edges, 1)
     height, _ = torch.min(edges, 1)
     return torch.stack([x_ctr, y_ctr, width, height, angles], 1)
+
+
+@FUNCTION_REWRITER.register_rewriter(
+    func_name='mmrotate.core.bbox.transforms.poly2obb_le135')
+def poly2obb_le135__default(ctx, polys):
```

Comment:
maybe we could add [rotated_retinanet_obb_r50_fpn_1x_dota_le135](https://github.com/open-mmlab/mmrotate/blob/main/configs/rotated_retinanet/rotated_retinanet_obb_r50_fpn_1x_dota_le135.py) in [mmrotate.yml ](https://github.com/open-mmlab/mmdeploy/blob/36c35b6e882cb082dd8d49a82a6a6f2d98b5b266/tests/regression/mmrotate.yml#L46)for this setting.

Reply:
It is already le135.